src/vapid-helper.js

```diff
@@ -90,6 +96,48 @@ function validatePrivateKey(privateKey) {
 }
 
 /**
+ * Calculates the total expiration by adding
+ * value of Date.now() with the seconds passed
+ *
+ * @param {Number} numSeconds Number of seconds to calculate expiration
+ * @return {Number} expiration in seconds
+ */
+function calculateExpiration(numSeconds) {
+  return Math.floor(Date.now() / 1000) + numSeconds;
+}
+
+/**
+ * Validates the Expiration based on the VAPID Spec
+ *
+ * `expiration` is not validated with a `minimum` value,
+ * since the time at validation is going to be different
+ * than at the time of creation of the custom expiration value
+ */
+function validateExpiration(expiration) {
+  if (!expiration) {
+    throw new Error('No expiration value provided in `expiration`');
+  }
+
+  expiration = Number.parseInt(expiration, 10);
+
+  if (isNaN(expiration) || typeof expiration !== 'number') {
+    throw new Error('Invalid expiration value passed, expiration must be `Number`');
```

Comment:
@gauntface Yup! This explicitly parses a string passed to it into a number, if it's not already a number. But since it's causing confusion, this can be removed, so the option only accepts number.